pkgs/top-level/python-packages.nix

```diff
@@ -4378,6 +4378,8 @@ self: super: with self; {
 
   homeassistant-pyozw = callPackage ../development/python-modules/homeassistant-pyozw { };
 
+  homeassistant-stubs = callPackage ../servers/home-assistant/stubs.nix { };
```

Comment:
I'd put it in
```suggestion
  homeassistant-stubs = callPackage ../development/python-modues/homeassistant-stubs { };
```

Reply:
I have a slight preference for keeping home-assistant packages more tightly grouped together, but not a strong opinion either.